source/release-notes.txt

```diff
@@ -10,6 +10,23 @@ Release Notes
    :depth: 1
    :class: twocols
 
+|compass| 1.43.4
```

Comment:
Let's also update the version in the snooty.toml file for this release.

Reply:
thanks for catching that! updating.